rxgroups/src/main/java/com/airbnb/rxgroups/ProcessorUtils.java

```diff
@@ -0,0 +1,59 @@
+package com.airbnb.rxgroups;
+
+
+import java.util.List;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.type.DeclaredType;
+import javax.lang.model.type.TypeKind;
+import javax.lang.model.type.TypeMirror;
+
+class ProcessorUtils {
+
+  static boolean isResubscribingObserver(Element observerFieldElement) {
```

Comment:
I can try that. I originally got this code from butterknife and have used it in other processors. I think this is necessary for handling types or something... but don't quite remember atm. I can try it.